README.md

```diff
@@ -498,6 +498,8 @@ An advanced `Agenda` component that can display interactive listings for calenda
   rowHasChanged={(r1, r2) => {return r1.text !== r2.text}}
   // Hide knob button. Default = false
   hideKnob={true}
+  // When `true` and `hideKnob` prop is `false`, the knob will always be visible and the user will be able to drag the knob up and close the calendar. Default = false
+  knobCanCloseCalendar={false}
```

Comment:
I don't really like the prop naming... How about something like `showClosingKnob`?

Reply:
Updated!